src/Sql/dbo/Stored Procedures/OrganizationSponsorship_DeleteExpired.sql

```diff
@@ -0,0 +1,19 @@
+CREATE PROCEDURE [dbo].[OrganizationSponsorship_DeleteExpired]
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    DECLARE @BatchSize INT = 100
+    DECLARE @Now DATETIME2(7) = GETUTCDATE()
```

Comment:
Do we want to include some leeway here? It seems like at _least_ a day for sync reasons is a good idea. Probably significantly more.

It probably makes sense to change this into a parameter so we can specify a leeway without a migration